internal/setup/setup.go

```diff
@@ -475,6 +487,12 @@ func parseRelease(baseDir, filePath string, data []byte) (*Release, error) {
 		if details.Default {
 			release.DefaultArchive = archiveName
 		}
+		switch details.Pro {
```

Comment:
If we end up using the map[proValue]... then let's use the map here as well to avoid having to update several places in the code.

Reply:
We did not end up doing that eventually.